content/blog/2025/05/2025-05-22-cfp-matrix-conf-25.md

```diff
@@ -0,0 +1,38 @@
++++
+date = "2025-05-22T13:00:00Z"
+title = "Announcing the Matrix Conference 2025"
+
+[taxonomies]
+author = ["Thib"]
+category = ["Events", "Conference"]
++++
+
+
+The [Matrix.org](http://Matrix.org) Foundation is happy to announce that the [Matrix Conference 2025](https://2025.matrix.org/) will be held at [Kaleidoscoop](https://www.kaleidos.coop/) in Strasbourg, France, from October 15 to 18!
+
+Last year the Conference was completely sold out, with more than 200 decision makers, experts and enthusiasts joining us for this very first edition! The rooms were so full, and we had such a long waiting list that we've decided to extend our capacity this year to 300 attendees, with flexibility for more.
+
+![](/blog/img/full-capacity.webp)
```

Comment:
https://github.com/matrix-org/matrix.org/blob/main/CONTENT.md#adding-pictures-in-your-post a11y please :)

Reply:
@thibaultamartin why not use the proper `figure` template?

Reply:
Further, I understand the idea of having the picture before `<more>` but maybe with this specific picture it's nicer to have it inside the post :thinking: